README.md

```diff
@@ -1,159 +1,170 @@
 
-[MarkLogic Server Documentation](https://docs.marklogic.com/10.0)
+Documentation for external projects, tools, and specifications referenced by this README are available as follows:
 
-[Everything About HL7/FHIR](https://www.hl7.org/fhir/index.html)
+- [MarkLogic Data Hub](https://docs.marklogic.com/datahub/5.5/index.html)
+- [MarkLogic Server](https://docs.marklogic.com/10.0)
+- [HL7/FHIR](https://www.hl7.org/fhir/index.html)
 
+## Get the Healthcare Starter Kit (HSK)
 
-## Get the Healthcare Starter Kit
+Clone the source or download a tagged release zip file from the [MarkLogic HSK](https://github.com/marklogic-community/marklogic-healthcare-starter-kit) repository.
 
-Clone the source or download the tagged release zip file from the MarkLogic Community GitHub repository:
+## Deploy the HSK
 
-[MarkLogic Health Care Starter Kit](https://github.com/marklogic-community/marklogic-healthcare-starter-kit).
-
-## Deploy the Healthcare Starter Kit
-
-This version (0.9.0) of the HSK project was built and tested with the following prerequisites:
+The HSK was built and tested with the following prerequisites:
 
 - Java 8 or 11
-- Gradle >= v6.4
-  - Note: we recommend running all commands with the built-in the Gradle wrapper function, `./gradlew`, which requires no installation.
-- MarkLogic Data Hub v5.5.1
+- [MarkLogic Data Hub Central v5.5.1](http://developer.marklogic.com/download/binaries/dhf/marklogic-data-hub-central-5.5.1.war)
 - MarkLogic Server >= v10.0-7
 
 ### Installation Steps:
 
-- [Download MarkLogic Data Hub Central v.5.5.1 web application archive (.war) file](http://developer.marklogic.com/download/binaries/dhf/marklogic-data-hub-central-5.5.1.war)
+> Note: Installation steps assume a MarkLogic Server user/role with sufficient privileges is specified. Refer to the MarkLogic Data Hub documentation if needed.
+
+- Download MarkLogic Data Hub Central using the link above
 - Unzip the tagged release or clone the source into a directory of your choosing.
-- At the top level of the `healthcare-starter-kit` directory, edit the `mlUsername` and `mlPassword` properties within the `gradle-local.properties` file, based on the MarkLogic user you intend to use. 
-  - Note: This assumes a MarkLogic Server user/role with sufficient privileges; please refer to MarkLogic Data Hub documentation if needed.
-  - The project includes several sample demo users, such as `DrSmith` (password `"demo"` sans quotes), who is capable of running all operations.
+- At the top level of your project directory, change the `mlUsername` and `mlPassword` properties in `gradle-local.properties`, based on the MarkLogic user you intend to use.

```

Comment:
We should clarify the username and password should be the username and password for the server admin user, or a similarly-provisioned, pre-existing user that is authorized to deploy and configure all aspects of the server. (I'm not sure what the minimal role is since we always use admin in development)